arch/arm/src/lpc54xx/lpc54_ethernet.c

```diff
@@ -934,6 +933,8 @@ static void lpc54_eth_reply(struct lpc54_ethdriver_s *priv)
 
 static void lpc54_eth_rxdispatch(struct lpc54_ethdriver_s *priv)
 {
+  struct net_driver_s *dev = &priv->eth_dev;
```

Comment:
This variable (`dev`) is not used in this function. Perhaps you intended to change all `&priv->eth_dev` below?